_sass/base.scss

```diff
@@ -5894,6 +5918,178 @@ th, td {
 
 }
 
+.half-block-tools {
```

Comment:
Intended css ?

Reply:
Yes it is, Although I havent committed the related HTML yet